### PR TITLE
[library] Added bar and map charts to experimental page

### PR DESCRIPTION
**Pull request by dwnoble**
dc-map-bar

### CODE REVIEW

File: static/js/components/tiles/line_tile.tsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ export const fetchData = async (props: LineTilePropType) => {
       statVars.push(spec.denom);
     }
   }
-  let endpoint = `${getRoot()}/api/observations/series`;
+  let endpoint = "/api/observations/series";
```
Comment (dwnoble) [inline static/js/components/tiles/line_tile.tsx:122]: Change this to apiRoot

File: experimental/sdg-static/js/index.js [hunk 2]
```diff
@@ -89,18 +89,42 @@ function renderBlocks() {
...
  */
 function renderChart($container, chartConfig) {
-  if (chartConfig.type == "LINE") {
+  if (chartConfig.type == "BAR") {
+    const chartElement = document.createElement("div");
+    $container.append(chartElement);
+    var lineChartProps = {
+      id: _.uniqueId("chart-"),
+      svgChartHeight: 200,
+      className: undefined,
```
Comment (shifucun) [inline experimental/sdg-static/js/index.js:98]: what is className used for?

**Review by shifucun — APPROVED** [review]
looks nice!

File: static/js/components/tiles/map_tile.tsx [hunk 3]
```diff
@@ -414,3 +415,15 @@ export function draw(
     );
   }
 }
+
+/**
+ * Renders map chart tile component in the given HTML element
+ * @param element DOM element to render the chart
+ * @param props line chart tile component properties
```
Comment (chejennifer) [inline static/js/components/tiles/map_tile.tsx:422]: map, not line?

File: static/js/components/tiles/bar_tile.tsx [hunk 4]
```diff
@@ -257,3 +258,15 @@ export function draw(
     chartData.unit
   );
 }
+
+/**
+ * Renders bar chart tile component in the given HTML element
+ * @param element DOM element to render the chart
+ * @param props line chart tile component properties
```
Comment (chejennifer) [inline static/js/components/tiles/bar_tile.tsx:265]: bar?

**Review by chejennifer — APPROVED** [review]
super cool!

File: experimental/sdg-static/js/index.js (hunk 2 above)
Comment (dwnoble) [reply experimental/sdg-static/js/index.js:98]: this lets users add a custom className (and thus to apply custom styles) to charts. It defaults to undefined, but is overridden if set in the sdg.json config file. I can add an example of this in a follow-up pr

File: static/js/components/tiles/map_tile.tsx (hunk 3 above)
Comment (dwnoble) [reply static/js/components/tiles/map_tile.tsx:422]: fixed! thank you

**Pull request merged**